lib/resque/durable.rb

```diff
@@ -66,10 +66,24 @@ def around_perform_manage_audit(*args)
           yield
         end
 
-        a.complete!
+        if re_enqueue_immediately?
```

Comment:
The first time I read these method names I grimaced at the aesthetics of them. I read https://english.stackexchange.com/questions/93846/is-re-enqueue-or-reenqueue-a-proper-word, and contemplated whether "reenqueue_immediately" or "requeue_immediately" was appropriate.

I sat on it for a couple hours and no longer care.